cwms-data-api/src/main/java/cwms/cda/api/Controllers.java

```diff
@@ -121,15 +121,22 @@ public final class Controllers {
     public static final String TIMESERIES = "timeseries";
     public static final String LOCATIONS = "locations";
 
+    public static final String SPEC_ID = "spec-id";
```

Comment:
I think this should be "name" to match the other similar fields.

Reply:
done